infra/app/app-config/env-config/main.tf

```diff
@@ -5,4 +5,20 @@ locals {
   prefix = terraform.workspace == "default" ? "" : "${terraform.workspace}-"
 
   bucket_name = "${local.prefix}${var.project_name}-${var.app_name}-${var.environment}"
+
+  database_config = var.has_database ? {
+    region                      = var.default_region
+    cluster_name                = "${var.app_name}-${var.environment}"
+    app_username                = "app"
+    migrator_username           = "migrator"
+    schema_name                 = var.app_name
+    app_access_policy_name      = "${var.app_name}-${var.environment}-app-access"
+    migrator_access_policy_name = "${var.app_name}-${var.environment}-migrator-access"
+
+    # Enable extensions that require the rds_superuser role to be created here
+    # See docs/infra/set-up-database.md for more information
+    superuser_extensions = {
+      "vector" : false,
```

Comment:
Is this a necessary default to include? If most projects won't need it, then I think it's better to let the object be empty `{}` and to include the example in the comments.

Reply:
Removed!